packages/site/src/components/CategoryCardSection.module.css

```diff
@@ -1,15 +0,0 @@
-.categoryCardSection {
-    display: flex;
-    flex-direction: column;
-    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
-    gap: var(--space-larger);
-    width: 100%;
-    padding-bottom: var(--space-large);
```

Comment:
Worth noting, I pulled this padding off when moving it to the CSS because the Content wrapping these should determine how much space there is between sections